autoparams/src/main/java/org/javaunit/autoparams/generator/DefaultObjectGenerator.java

```diff
@@ -9,7 +9,10 @@ public DefaultObjectGenerator() {
             new SimpleValueObjectGenerator(),
             new CollectionGenerator(),
             new StreamGenerator(),
-            new ComplexObjectGenerator()
+            new RecursiveObjectGuard(
+                new ComplexObjectGenerator(),
+                3
```

Comment:
Named parameters are still not supported in the 21st century...😱

Reply:
![image](https://user-images.githubusercontent.com/973743/132115080-c284a0b8-0417-4d3b-a608-1bd8f4b0f222.png)